runtime-attach/build.gradle.kts

```
@@ -11,4 +11,15 @@ dependencies {
 
   // Used by byte-buddy but not brought in as a transitive dependency.
   compileOnly("com.google.code.findbugs:annotations")
+
+  testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent:1.13.1")
```

probably we should add to `dependencyManagement/build.gradle.kts` to resolve version there

```suggestion
  testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent:1.14.0")
```